aspen/http/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+:mod:`http`
+===========
+
+Aspen doesn't implement all of HTTP, only things related to filesystem
+dispatch.
+
+.. automodule:: aspen.http.mapping
+.. automodule:: aspen.http.request
+.. automodule:: aspen.http.resource
+
+"""
```

Comment:
Querystring is still used by the JSONP renderer. We'll also need to import Method from pando to be able to implement the pure-python alternatives to simplates, because they use HTTP methods as function names.